src/storm/solver/helper/DiscountedValueIterationHelper.cpp

```diff
@@ -0,0 +1,150 @@
+#include "storm/solver/helper/DiscountedValueIterationHelper.h"
+
+#include "storm/adapters/RationalNumberAdapter.h"
+#include "storm/solver/helper/ValueIterationOperator.h"
+#include "storm/utility/Extremum.h"
+
+namespace storm::solver::helper {
+
+template<typename ValueType, storm::OptimizationDirection Dir, bool Relative>
+class DiscountedVIOperatorBackend {
+   public:
+    DiscountedVIOperatorBackend(ValueType const& precision, ValueType const& discountFactor, ValueType const& maximalAbsoluteReward)
+        : precision{precision},
+          discountFactor{discountFactor},
+          bound{(((storm::utility::one<ValueType>() - discountFactor) * precision) / (2 * discountFactor))} {
```

Comment:
I am slightly confused by the factor two here? Can we add a comment that clarifies this bound variable? :-)

Reply:
I checked and the factor 2 is indeed incorrect, thanks for spotting that! I added a reference to the Russell/Norvig book